<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DataPlotWidget</class>
    <widget class="QWidget" name="DataPlotWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>533</width>
                <height>408</height>
            </rect>
        </property>
        <property name="acceptDrops">
            <bool>true</bool>
        </property>
        <property name="windowTitle">
            <string>Plot</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
                <layout class="QGridLayout" name="gridLayout_3">
                    <property name="sizeConstraint">
                        <enum>QLayout::SetDefaultConstraint</enum>
                    </property>
                    <item row="1" column="1">
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <property name="spacing">
                                <number>6</number>
                            </property>
                            <property name="sizeConstraint">
                                <enum>QLayout::SetDefaultConstraint</enum>
                            </property>
                            <item>
                                <layout class="QGridLayout" name="gridLayout_4">
                                    <item row="0" column="2">
                                        <widget class="QPushButton" name="clear_button">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="toolTip">
                                                <string>clear plot</string>
                                            </property>
                                            <property name="iconSize">
                                                <size>
                                                    <width>16</width>
                                                    <height>16</height>
                                                </size>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="0">
                                        <widget class="QPushButton" name="btn_setting">
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="2" column="0" colspan="3">
                                        <widget class="QCheckBox" name="cbox_autoscale_y">
                                            <property name="text">
                                                <string>auto scale y</string>
                                            </property>
                                            <property name="checked">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="0" colspan="3">
                                        <widget class="QCheckBox" name="autoscroll_checkbox">
                                            <property name="text">
                                                <string>autoscroll</string>
                                            </property>
                                            <property name="checked">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="1">
                                        <widget class="QPushButton" name="pause_button">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="toolTip">
                                                <string>pause plot</string>
                                            </property>
                                            <property name="iconSize">
                                                <size>
                                                    <width>16</width>
                                                    <height>16</height>
                                                </size>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout"/>
                            </item>
                            <item>
                                <layout class="QFormLayout" name="formLayout">
                                    <item row="0" column="0">
                                        <widget class="QLabel" name="yMaxLabel">
                                            <property name="text">
                                                <string>YMax</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="1">
                                        <widget class="QSpinBox" name="yMaxSpinBox">
                                            <property name="minimum">
                                                <number>-32767</number>
                                            </property>
                                            <property name="maximum">
                                                <number>32767</number>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="0">
                                        <widget class="QLabel" name="yMinLabel">
                                            <property name="text">
                                                <string>YMin</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="1">
                                        <widget class="QSpinBox" name="yMinSpinBox">
                                            <property name="minimum">
                                                <number>-32767</number>
                                            </property>
                                            <property name="maximum">
                                                <number>32767</number>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_showall">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>200</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Show all</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_hideall">
                                    <property name="maximumSize">
                                        <size>
                                            <width>200</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Hide all</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QListWidget" name="listWidgetChannels">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>200</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item row="1" column="0">
                        <layout class="QVBoxLayout" name="plotLayout"/>
                    </item>
                </layout>
            </item>
            <item row="0" column="0">
                <layout class="QHBoxLayout" name="dataPlotControls">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <property name="leftMargin">
                        <number>5</number>
                    </property>
                    <property name="topMargin">
                        <number>5</number>
                    </property>
                    <property name="rightMargin">
                        <number>5</number>
                    </property>
                    <property name="bottomMargin">
                        <number>5</number>
                    </property>
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>Port</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="edit_port">
                            <property name="text">
                                <string>/dev/ttyUSB2</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Baudrate</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="edit_baudrate">
                            <property name="text">
                                <string>230400</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_open">
                            <property name="text">
                                <string>Open</string>
                            </property>
                            <property name="iconSize">
                                <size>
                                    <width>16</width>
                                    <height>16</height>
                                </size>
                            </property>
                            <property name="checkable">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_save">
                            <property name="text">
                                <string>Save</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="2" column="0">
                <widget class="QLabel" name="label_status">
                    <property name="text">
                        <string/>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
